src/admin.rs

```diff
@@ -368,26 +368,39 @@ fn list_loggers() -> Response<Full<Bytes>> {
     }
 }
 
-fn change_log_level(reset: bool, level: &str) -> Response<Full<Bytes>> {
-    match tracing::level_filters::LevelFilter::from_str(level) {
-        Ok(level_filter) => {
-            // Valid level, continue processing
-            tracing::info!("Parsed level: {:?}", level_filter);
-            match telemetry::set_level(reset, level) {
-                Ok(_) => list_loggers(),
-                Err(e) => plaintext_response(
-                    hyper::StatusCode::BAD_REQUEST,
-                    format!("Failed to set new level: {}\n{}", e, HELP_STRING),
-                ),
+fn validate_log_level(level: &str) -> anyhow::Result<()> {
+    for clause in level.split(',') {
+        // We support 2 forms, compared to the underlying library
+        // <level>: supported, sets the default
+        // <scope>:<level>: supported, sets a scope's level
+        // <scope>: sets the scope to 'trace' level. NOT SUPPORTED.
```

Comment:
Where do we document the scopes?

Reply:
We don't unfortunately, they are all basically the rust module name

Reply:
:+1:  Might be good to add an admin endpoint that dumps the available/registered scopes (if that's possible, I imagine it is) eventually.